examples/draw_turtle.rs

```diff
@@ -0,0 +1,314 @@
+extern crate turtle;
+
+use turtle::Turtle;
+
+const SIZE: f64 = 1.0;
+const FIRST_COLOR: [f64; 4] = [62.0, 114.0, 29.0, 1.0];
+const SECOND_COLOR: [f64; 4] = [119.0, 178.0, 85.0, 1.0];
+const EYE_COLOR: &str = "black";
```

Comment:
Could you make the type of these color constants `Color`? Rust won't allow you to use `into()` in constants, so you can use [`color::BLACK`](https://docs.rs/turtle/1.0.0-alpha.8/turtle/color/constant.BLACK.html) for the eye color. You will need to change your `use` statement to be `use turtle::{Turtle, Color, color}`.